bitmap-raziyehLogan/model/bitmap.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = (buff) => {
```

Comment:
@loganlsr @Raziyehbazargan - this looks good - that said, it's standard practice to keep with the constructor pattern and without using lexical arrow functions:

```
const Bitmap = module.exports = function(buff) {
  \\ properties go here
}
```